### PR TITLE
Set up book-list

### PR DESCRIPTION
**Pull request by bohaz**
> ## In this pull request

- [ ] Setup a new React application using [Create React App](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app)
- [ ] Create a directory for your reusable components: `/components`.
- [ ] Install [React Router V6](https://reactrouter.com/en/main/start/tutorial#setup)
- [ ] [Setup your router](https://reactrouter.com/en/main/start/tutorial#adding-a-router) inside of `<App>`
- [ ] Add 2 routes in your application, each should render a page component:
`/`
`/categories`
- [ ] Create components inside of the `/components` directory:
- Contains individual book state, as well as a button to delete the book
- Contains a list to renders individual books
- Contains a form to create new books
- Contains a navigation to link to the 2 routes you created

### CODE REVIEW

**Review by LionRouge1 — COMMENTED** [review]
Hi @bohaz, 

done

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project that doesn't have a professional README file. Kindly use the recommended Microverse [README template](https://github.com/microverseinc/readme-template) and resubmit your project for review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/App.js [hunk 1]
```diff
@@ -1,24 +1,29 @@
...
+          </li>
+          <li>
+            <Link to="/categories">Categories</Link>
+          </li>
+        </ul>
+      </nav>
+
+      <Routes>
+        <Route exact path="/" component={HomePage} />
+        <Route path="/categories" component={CategoriesPage} />
```
Comment (AlphaNtihinduka) [inline src/App.js]: - [ ] You have done great to ensure that the Navbar contains two links. However, as the project requirements you should ensure that the Home page displays Books list and book form ☺️.

Your design ☺️ 
image

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @bohaz 🙋‍♂️
Good job working on bookstore. You have done the design to the best expectation and incredibly well. 👏  

You still need to implement few on the project but you are almost there!!! 🚶: 

TO HIGHLIGHT:
- Correct use of react-router-dom
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

File: src/App.js (hunk 1 above)
Comment (bohaz) [reply src/App.js]: Thanks!! I appreciate your professional review. 
DoneCongratsGIF

**Review by Gambit142 — APPROVED** [review]
Hi @bohaz 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linters checks passed ✔️ 
- Installed `react-router-dom` ✔️ 
- Two routes were added to the application ✔️ 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**